api.md

```diff
@@ -2775,6 +2775,8 @@ Specifically this sets the `timescaledb.restoring` GUC to `on` and stops any
 background workers which may have been performing tasks until the [`timescaledb_post_restore`](#timescaledb_post_restore)
 fuction is run following the restore. See [backup/restore docs][backup-restore] for more information.
 
+>:WARNING: Using this function when doing an upgrade could cause issue in previous versions of TimescaleDB.
```

Comment:
Nit: could cause an issue, or could cause issues, not could cause issue.

Reply:
Used "could cause issues".